conda-recipe/meta.yaml

```diff
@@ -22,9 +22,10 @@ requirements:
       - entrypoints
       - jinja2
       - prettytable
-      - psdm_qs_cli
-      - pymongo
       - simplejson
+    run-constrained:
+      - psdm_qs_cli >=0.3.1
+      - pymongo >=4.0.2
```

Comment:
I think something like `extras` configuration is more appropriate for optional dependencies- I don't remember exactly how to do this (not a conda thing).

Requiring a recent API for these makes it easier on us as we move forward, so it's probably correct to do this :+1:. The exact version here doesn't matter too much. We can update these exact numbers as appropriate if we use newer features of these dependencies.